uid2_client/refresh_keys_util.py

```diff
@@ -0,0 +1,70 @@
+import datetime as dt
+import json
+from datetime import timezone
+
+from .keys import EncryptionKey, EncryptionKeysCollection
+from .request_response_util import *
+from .identity_scope import IdentityScope
+
+
+def _make_dt(timestamp):
+    return dt.datetime.fromtimestamp(timestamp, tz=timezone.utc)
+
+
+def parse_keys_json(resp_body):
+    keys = []
+    identity_scope = IdentityScope.UID2
+    for key in resp_body["keys"]:
+        keyset_id = None
+        if "keyset_id" in key:
+            keyset_id = key["keyset_id"]
```

Comment:
Can we do

```
keyset_id = key.get("keyset_id")
```

Reply:
Interesting. I didn't know this field was optional in the response. https://github.com/IABTechLab/uid2-operator/blob/993e2c56587dc302f582497e87a06c2f802377f6/src/main/java/com/uid2/operator/vertx/UIDOperatorVerticle.java#L554-L558

Reply:
yes, that's more elegant